apps/video/index.html

```diff
@@ -64,59 +64,79 @@ <h1 id="thumbnails-number-selected"></h1>
-          <a id="thumbnails-share-button" class="button" data-icon="share"></a>
+          <a id="thumbnails-delete-button" class="button" data-icon="delete"
+             role="button" data-l10n-id="delete-button"></a>
+          <a id="thumbnails-share-button" class="button" data-icon="share"
+             role="button" data-l10n-id="share-button"></a>
         </footer>
       </section>
 
       <!-- (maximized) video player -->
       <section role="region" id="player-view">
         <!-- display this at startup while we create thumbnails -->
-        <div id="spinner-overlay" class="hidden"><div id="spinner"></div></div>
-        <div id="video-container">
-          <video src="about:blank" id="player"></video>
+        <div id="spinner-overlay" class="hidden" role="progress"
+             data-l10n-id="spinner-overlay"><div id="spinner"></div></div>
+        <!-- video controls header -->
+        <gaia-header id="player-header" action="back" class="video-controls">
+          <h1 id="video-title"></h1>
+          <button id="options" data-icon="more"
+                  data-l10n-id="options-more"></button>
+          <button id="picker-done" data-l10n-id="done">Done</button>
+        </gaia-header>
+        <div id="video-container" role="button">
+          <!-- Hiding the video element from the screen reader since the
+               controls are implemented elsewhere. -->
+          <video src="about:blank" id="player" aria-hidden="true"></video>
         </div>
-        <!-- video controls -->
-        <div id="videoControls">
-          <section role="region">
-            <gaia-header id="player-header" action="back">
-              <h1 id="video-title"></h1>
-              <button id="options" data-icon="more"></button>
-              <button id="picker-done" data-l10n-id="done">Done</button>
-            </gaia-header>
-            <footer id="videoBar">
-              <div id="timeSlider">
-                <span id="elapsed-text"></span>
-                <div id="slider-wrapper">
-                  <div id="elapsedTime" class="progress"></div>
-                  <div id="bufferedTime" class="progress"></div>
-                  <div id="timeBackground" class="progress"></div>
-                  <button id="playHead"></button>
-                </div>
-                <span id="duration-text"></span>
-              </div>
-              <div id="fullscreen-button"></div>
-            </footer>
-            <footer id="videoControlBar">
-              <div id="videoToolBar">
-                <button id="seek-backward" class="player-controls-button" data-icon="skip-back"></button>
-                <button id="play" class="player-controls-button" data-icon="pause"></button>
-                <button id="seek-forward" class="player-controls-button" data-icon="skip-forward"></button>
+        <!-- video controls footer toolbar -->
+        <div role="toolbar" class="video-controls video-controls-bottom">
```

Comment:
I understand the motivation in moving the 'gaia-header' element so that the order of the elements is consistent with the presentation. However, I'm not able to see the motivation for removing the 'videoControls' element and replacing it with a different div element. Why not add the 'toolbar' role and 'video-controls' class to the existing videoControls element? That seems like less of a change, i.e., simpler.